run_project_tests.py

```diff
@@ -282,6 +300,8 @@ def __lt__(self, other: object) -> bool:
 backend:      'Backend'
 backend_flags: T.List[str]
 
+python_info: 'PythonIntrospectionDict'
```

Comment:
https://bugs.python.org/issue34939

cross-only-armhf fails, but not in my fork due to never running :/ and this is a lovely python bug in 3.7